helmfile/50-applications.yaml

```diff
@@ -616,7 +616,7 @@ releases:
   - values/fluentd/log-manager.yaml.gotmpl
 
 - name: rclone-sync
-  namespace: kube-system
+  namespace: rclone-sync
```

Comment:
Not a super strong opinion on this but maybe simply using the namespace `rclone` is sleeker ;)